discovery/manager_test.go

```diff
@@ -772,9 +772,9 @@ func verifyPresence(t *testing.T, tSets map[poolKey]map[string]*targetgroup.Grou
 	if match != present {
 		msg := ""
 		if !present {
-			msg = "not"
+			msg = " not "
 		}
-		require.FailNow(t, "%q should %s be present in Targets labels: %q", label, msg, mergedTargets)
+		require.FailNow(t, fmt.Sprintf("%q should%sbe present in Targets labels: %q", label, msg, mergedTargets))
```

Comment:
ditto

Reply:
Also we might need to add space to `msg` or remove next to `" not"` and update as follows:

```suggestion
		require.FailNow(t, fmt.Sprintf("%q should %sbe present in Targets labels: %q", label, msg, mergedTargets))
```

Reply:
@bwplotka Spaces are already in `msg` though?

```go
msg := ""
if !present {
	msg = " not "
}
```

Reply:
Good catch, thanks!